feedhenry/fh-ios-sdk
# Transitioning from OCUnit to XCTest

Motivation
The ios-sdk uses the old/deprecated OCUnit test framework. We should transition to the newer XCTest framework to allow future tests to be written within.

To tests:
Simply open ```fh-ios-sdk.xcodeproj``` and run ```Product->Tests```. The project should build and tests run successfully.

@wei-lee mind to review?
